games/o.yaml

```diff
@@ -1800,6 +1800,36 @@
   url: https://openxcom.org/
   updated: 2020-04-09
 
+- name: OpenXRay
+  originals:
+  - S.T.A.L.K.E.R.
+  type: remake
+  repo: 'https://github.com/OpenXRay/xray-16'
+  development: very active
+  status: playable
+  multiplayer:
+  - Online
+  - LAN
+  content: commercial
+  lang:
+  - C++
+  - Lua
+  framework:
+  - OpenXRay
+  - X-Ray
```

Comment:
remove OpenXRay and X-Ray - this field should be engines/frameworks that the game is developed using

Reply:
Done.